src/DependencyInjection/Configuration.php

```diff
@@ -188,8 +183,8 @@ public function getConfigTreeBuilder()
                                 ->ifTrue()
                                 ->then(static function (bool $v): bool {
                                     @trigger_error(
-                                        'Using `true` as value for "sonata_admin.options.legacy_twig_text_extension" option is deprecated since sonata-project/admin-bundle 3.64. '.
-                                        'You should set it to `false`, which will be the default value since version 4.0.'
+                                        'Using `true` as value for "sonata_admin.options.legacy_twig_text_extension" option is deprecated since sonata-project/admin-bundle 3.64 and will be remove in 4.0'.
+                                        'You should set it to `false` before upgrade process.'
```

Comment:
```suggestion
                                        'Using `true` as value for "sonata_admin.options.legacy_twig_text_extension" option is deprecated since sonata-project/admin-bundle 3.64 and will be removed in version 4.0.'.
                                        ' Remove it or set its value to `false`.'
```

Reply:
At first it should be set to flase, then can be delete in upgrade process becouse by default is it true. Remove it is bad idea